apps/planning-suite-kit/package.json

```diff
@@ -12,13 +12,13 @@
     "test": "TRUFFLE_TEST=true npm run ganache-cli:test"
   },
   "dependencies": {
-    "@aragon/apps-finance": "^3.0.0",
-    "@aragon/apps-shared-minime": "^1.0.1",
-    "@aragon/apps-survey": "^1.0.0-rc.2",
-    "@aragon/apps-token-manager": "^2.0.0",
-    "@aragon/apps-voting": "^2.0.0",
-    "@aragon/apps-vault": "^4.1.0",
-    "@aragon/os": "^4.2.0",
+    "@aragon/apps-finance": "3.0.0",
+    "@aragon/apps-shared-minime": "1.0.1",
+    "@aragon/apps-survey": "1.0.0-rc.2",
+    "@aragon/apps-token-manager": "2.0.0",
+    "@aragon/apps-voting": "2.0.0",
+    "@aragon/apps-vault": "4.1.0",
+    "@aragon/os": "4.2.0",
```

Comment:
This is all good stuff but probably unrelated to getting Address Book working. If that's so, please move this to a new PR.